NewForce-Cohort-9/newforce-c9-tabloidmvc-TabloidMVC
# #20 Create a Comment

# Description
When a user is viewing the details of a Post, they are able to add a comment to the post. [Trello Ticket #20](https://trello.com/c/9LfQxLQV/20-20-create-a-comment)

## Type of change


# Testing Instructions

- From the command line:
  - `git add` and `git commit` whatever you're working on on your branch
  - `git checkout main`
  - `git fetch --a`
  - `git checkout il-comment-2`
  - If you don't see the most recent version of my files, `git pull origin il-comment-2`
- If you haven't created the Tabloid MVC database, please do so.
  - Use this to fill out tables and columns: [SQL data](https://github.com/NewForce-Cohort-9/tabloidmvc-dalvin-and-the-chipmunks/blob/main/SQL/01_TabloidMVC_Create_DB.sql)
  - Use this to seed the database: [SQL Seed Data](https://github.com/NewForce-Cohort-9/tabloidmvc-dalvin-and-the-chipmunks/blob/main/SQL/02_TabloidMVC_Seed_Data.sql)
- Click the execute with debugger solid green button to launch the app
- In browser, please go to Login on the nav bar and log in with this email: admin@example.com
- This should redirect you back to the home page.
- Select 'Posts' in the navbar and be directed to the list of all posts.
- Select view on the first post titlted 'C# is the  Best Language :)" and be directed to the post's details page. 
- Select 'Add Comment' and be directed to the comment form.
- Input a subject ("test") and content ("testing pr") for the comment
- Select 'Create' and you should be directed to a  list of all comments for the post.

# Checklist:
